ci/azure-pipelines/jobs/bazel-build-serving-tensorrt.yml

```diff
@@ -23,7 +23,8 @@ jobs:
                     -c opt \
                     --disk_cache="$BAZEL_BUILD_CACHE" \
                     --action_env=LIBRARY_PATH=/usr/local/cuda-11.0/lib64/stubs \
-                    --incompatible_use_specific_tool_files=false
+                    --incompatible_use_specific_tool_files=false \
+                    --copt="-Ithird_party/cuda/include"
```

Comment:
Need "-Ithird_party/cuda/include" ? See adlik_serving/runtime/tensorrt/model/BUILD

Reply:
Without this config, bazel cannot build it successfully. Only adding in "adlik_serving/runtime/tensorrt/model/BUILD" does not work.